src/AppBundle/Controller/Api/FormRequestController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace AppBundle\Controller\Api;
+
+use AppBundle\Entity\FormRequest;
+use AppBundle\Entity\FormRequestType;
+use Mcfedr\JsonFormBundle\Controller\JsonController;
+use AppBundle\Exception\JsonHttpException;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/form_request", name="form_requests")
+ */
+class FormRequestController extends JsonController
+{
+    /**
+     * @Route("/{status}", requirements={"status": "pending|past"})
+     * @Method("GET")
+     *
+     * @return JsonResponse
+     */
+    public function listAction(Request $request, $status)
+    {
+        $em = $this->getDoctrine()->getManager();
+        $requestForms = $this->get('knp_paginator')
+            ->paginate(
+                $em->getRepository(FormRequest::class)->findBy([
```

Comment:
$em used only once, you can do like so $this->getDoctrine()->getRepository(FormRequest::class)->findBy... instead